thatsmybis/thatsmybis
# Support export of wishlist and loot priority data for Gargul

Gargul is a loot master addon that now also supports thatsmybis. Full documentation of what it currently supports and how to use it can be found on [curseforge](https://www.curseforge.com/wow/addons/gargul) or more specifically the [GitHub wiki](https://github.com/papa-smurf/Gargul/wiki)

**How to test:**
Navigate to Guild -> Export, scroll down to the Gargul section and click the `Download` button. Make sure everything works.

**Notes:**
- I wasn't too sure about the `throw \Exception` (couldn't find any Exception classes)
- I added the Gargul logo using an inline image which is suboptimal but I'll leave that to you

The queries used by the export methods seem very lightweight and well-optimized on my end, but by all means tweak them however you see fit.

This is what it looks like:

![image](https://user-images.githubusercontent.com/4224975/125429073-66015d3c-13cb-4cb4-8dc5-e2a89a8d0811.png)
